client/src/pages/home/HomePage.js

```diff
@@ -1,19 +1,61 @@
-import React from "react";
+import React, { useState } from "react";
 import ProverbList from "./../proverb/ProverbList";
-
+import AddProverb from "./../proverb/AddProverb";
 import Section from "./../../components/Section";
-import Breadcrumb from './../../components/Breadcrumb'
+import Breadcrumb from "./../../components/Breadcrumb";
+import Modal from "./../../components/Modal";
+import Button from "./../../components/Button";
+
+import './HomePage.css'
 
 const HomePage = () => {
+  //Modal States
+  const [modal, setModal] = useState({
+    isOpen: false,
+    type: "",
+    id: "",
+  });
+
+  //Modal Handlers
+
+  const handleShow = (type) => async (e) => {
+    e.preventDefault();
+    if (type === "add") {
+      setModal({
+        isOpen: true,
+        type: "Add",
+      });
+    }
+  };
   return (
     <div>
-   <Section
+      <Modal
+        isOpen={modal.isOpen}
+        modalClose={() => setModal({ isOpen: false })}
```

Comment:
```suggestion
        modalClose={() => setModal({ isOpen: false, type: undefined })}
```


I think we should reset the state here as well, so that when we add other modal types (for example Edit), then we wont accidentally open the add one